pmlc/conversion/stdx_to_llvm/stdx_to_llvm.cc

```diff
@@ -85,6 +97,25 @@ struct FPToUILowering : public LLVMLegalizationPattern<stdx::FPToUIOp> {
   }
 };
 
+struct ASinLowering : public LLVMLegalizationPattern<stdx::ASinOp> {
```

Comment:
I think we can have a single generic lowering class to handle all of these calls to runtime functions.